integrations/database/postgresql/kubearchive.sql

```diff
@@ -67,7 +67,8 @@ SET default_table_access_method = heap;
 
 CREATE TABLE public.log_url (
     uuid uuid NOT NULL,
-    url text NOT NULL
+    url text NOT NULL,
+    container_name text NOT NULL
```

Comment:
can we also add created_at and updated_at like we have in the `resource` table?

Reply:
sure

Reply:
done